tests/testthat/test_biome.R

```diff
@@ -1,14 +1,23 @@
 context("Hector with multiple biomes")
 
 ssp245 <- function() {
-  newcore(system.file("input", "hector_ssp245.ini",
-    package = "hector"
-  ),
-  name = "test core",
-  suppresslogging = TRUE
-  )
+    core <- newcore(system.file("input", "hector_ssp245.ini", package = "hector"),
+            name = "test core", suppresslogging = TRUE)
+
+    # names <- c(NPP_FLUX0(), BETA(), Q10_RH(), ECS(), DIFFUSIVITY(), AERO_SCALE())
```

Comment:
@bpbond something that I find somewhat curious/alarming is that if we initiate the hector core with the old hector parameter values more of the tests pass. But wouldn't we expect the sum(biomes) = global regardless of the parameters used?

Reply:
Ugh, that doesn't sound good. I can look at this in more detail later.